ext/node/lib.rs

```diff
@@ -426,8 +426,9 @@ deno_core::extension!(deno_node,
     "path/_constants.ts",
     "path/_interface.ts",
     "path/_util.ts",
+    "path/_posix.ts",
```

Comment:
Could someone enlighten me on what this code does and why I needed to add these lines here?

I'm new to deno internals.

Reply:
This is telling the build process which files need to be available to generate the final Deno binary. We list these files explicitly so that we are aware which files are gonna be loaded and used in the build step. Since you added `path/_posix.ts` in this PR, you needed to add list this file here.